src/libraries/System.Formats.Cbor/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
```

Comment:
Why the MS key, instead of the Open key? I think the MS key is only used for packages that are partial facades for .NET Framework.

Reply:
Fixed by removing Directory.Build.props altogether.